src/Text/OrderType/OrderType.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import { PlaneDeparture } from '@styled-icons/fa-solid/PlaneDeparture';
+import { Utensils } from '@styled-icons/fa-solid/Utensils';
+import { Paragraph } from '../Paragraph';
+
+const PLANE_ICON = PlaneDeparture;
+const UTENSILS_ICON = Utensils;
+
+export enum OrderTypeIdentifier {
+    EAT_IN = 'EAT_IN',
+    TAKE_OUT = 'TAKE_OUT',
+    DELIVERY = 'DELIVERY',
+    SELF_DELIVERY = 'SELF_DELIVERY',
+}
+
+const orderTypes = {
+    EAT_IN: 'Eat In',
+    TAKE_OUT: 'Take Out',
+    DELIVERY: 'Delivery',
+    SELF_DELIVERY: 'Self Delivery',
+};
+
+export interface IOrderTypeProps {
+    /* Type of order (EAT_IN, TAKE_OUT, DELIVERY, SELF_DELIVERY) */
+    orderType: OrderTypeIdentifier;
+}
+
+export const OrderType: React.FC<IOrderTypeProps> = ({
+    orderType,
+    ...props
+}): React.ReactElement => {
+    /* Returns order type with no underscores */
+    const handleOrderType = () => {
+        switch (orderType) {
+            case OrderTypeIdentifier.EAT_IN:
+                return orderTypes.EAT_IN;
```

Comment:
Why not just return the enum!?

Reply:
How would I remove the underscore then? Should I just reuse the regex from the terminal repo and create a separate helper function for it?

Reply:
Yep I would use a helper function. Becuase your basically doing the same thing with that object but it's excessive if you think about how many times you might have to do this. 
1 Function versus hundreds of objects.